src/components/cms/AuthorSkills/AuthorSkills.js

```diff
@@ -159,13 +142,7 @@ class AuthorSkills extends Component {
 
     return (
       <React.Fragment>
-        <DialogTitle>
-          <CloseButton onClick={actions.closeModal} />
-          Skills by {author}{' '}
-          <a href={authorUrl && authorUrl !== '<author_url>' ? authorUrl : '/'}>
-            <GithubAvatarImage alt={'GitHub'} src={githubAvatarSrc} />
```

Comment:
Why was the author name/url section removed? Add it.

Reply:
In this he said author name should not be loaded : https://github.com/fossasia/susi.ai/pull/2325#pullrequestreview-253098948